ten_quick_tips_live_coding.md

```diff
@@ -53,46 +53,46 @@ university courses involving the teaching of programming or related techniques e
 
 There is a limited body of research on the effectiveness of Live coding in programming education.
 Most studies focus on non-participatory live coding,
-demonstrating the programming process and opposing this to the use of static code on slides
-(https://doi.org/10.1145/3279720.3279725 and references therein, as well as https://dl.acm.org/doi/10.1145/3373165.3373182).
-So far, the technique show that live-coding is as good as if not better than static code examples.
+demonstrating the programming process and contrasting this with the use of static code on slides
+(see https://doi.org/10.1145/3279720.3279725 and references therein, as well as https://dl.acm.org/doi/10.1145/3373165.3373182).
+So far, the technique show that live-coding is as good as if not better than using static code examples.
 
 The ten quick tips described below are aimed at those interested in applying the technique in to their own teaching.
-These complement the "Ten quick tips for delivering programming lessons"
+They are meant to complement the "Ten quick tips for delivering programming lessons"
 https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007433.
 
+
 ## Tip 1. Go slowly.
 
 Say out loud what you are doing while you do it
 for every command you type,
 every word of code you write,
-and every menu item or website button you click,
-and then point to the command and its output on the screen and optionally go through it a second time.
+and every menu item or website button you click.
+Then, point to the command and its output on the screen
+and optionally go through it a second time.
```

Comment:
remove "optionally".

Reply:
This allows learners to catch up and check their understanding: if they are watching you type and trying to type the same code in themselves, they don't have time to _think_ about what they're doing. As you go through it a second time, they can check their understanding and correct any small typos they may have made. This is particularly important for learners who may not be fluent in the language of instruction, or who may have hearing, vision, or mobility impairments.

Reply:
+1